column_transforms/new_cast/new_cast.yaml

```diff
@@ -0,0 +1,20 @@
+name: new_cast
+description: |
+  Cast selected columns to a new type
+arguments:
+  casts:
+    type: cast_dict
```

Comment:
do we need a new type here? if we're gonna start using k-v more thoroughly in UDTs, should we just have a dict type?

Reply:
good call i forgot about the new column_value_dict just for these types of things